composer.json

```diff
@@ -29,22 +25,15 @@
         "loophp/phposinfo": "^1.7.1",
         "symfony/yaml": "^4.4 || ^5 || ^6"
     },
+    "conflict": {
+        "acquia/blt-multisite": "*"
+    },
```

Comment:
Don't add this conflict as of now, will check first if needed will be added as part of multisite implementation ticket